tools/git-hooks/config.sample.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+################################################################################
+##  Copyright 2022 leonardodalinky(linkyy2000313@gmail.com)
```

Comment:
this license header is not accpetable... All files  in ASF project should have the same license header.